.travis.yml

```diff
@@ -18,7 +18,7 @@ notifications:
   email: false
 
 after_success:
-  - julia -e 'if VERSION < v"0.7-"; cd(Pkg.dir("MathematicalSets")); else; import Pkg; end; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder()); Codecov.submit(process_folder())'
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
```

Comment:
Same comment, do `codecov: true` at the beggining of the file after `language`

Reply:
Same comment :wink: